package.json

```diff
@@ -110,6 +111,30 @@
         "onCommand:extension.brightscript.sendRemoteText"
     ],
     "contributes": {
+        "views": {
+            "debug": [
+                {
+                    "id": "rendezvousView",
+                    "name": "Rendezvous",
+                    "when": "debugType == 'brightscript'",
+                    "group": "Rendezvous"
+                }
+            ]
+        },
+        "menus": {
+            "view/title": [
+                {
+                    "command": "extension.brightscript.toggleSortingMethod",
+                    "when": "debugType == 'brightscript'",
+                    "group": "Rendezvous"
+                },
+                {
+                    "command": "extension.brightscript.toggleSortingAscDesc",
```

Comment:
Let's call this `extension.brightscript.toggleRendezvousSortDirection` or `extension.brightscript.rendezvous.toggleSortDirection`

Reply:
Ok, I think I am going to add a clear command as well under the command `extension.brightscript.rendezvous.clearHistory` as per @airving10 comment. This would be a very simple command to add and I think it is very worth adding.

Reply:
Done